### PR TITLE
Reorganize the Hosted CE request doc (SOFTWARE-4350)

### PR DESCRIPTION
**Pull request by brianhlin**
@lmichael107 @jdost321 we wanted to add a step-by-step overview of the Hosted CE process (https://opensciencegrid.atlassian.net/browse/SOFTWARE-4350) so I reorganized and added sections to make the process clearer in the table of contents:

hosted-ce-process

A bunch of other stuff was out of date so this turned into a full doc review, too. Let me know what you think.

### CODE REVIEW

**Comment by brianhlin** [comment]
Partially rendered preview here: https://github.com/opensciencegrid/docs/blob/d31b2c113da59817962b657b59a7280830392960/docs/compute-element/hosted-ce.md

File: docs/compute-element/hosted-ce.md [hunk 1]
```diff
@@ -1,157 +1,185 @@
...
 
--   Configure the Hosted CE with your system details
+After preparing your local cluster, apply for a Hosted CE by filling out the
+[cluster integration questionnaire](https://docs.google.com/forms/d/e/1FAIpQLSexKMFho_TGJ8nOY-qLXJf_8neAnjDSJqrNbYIUvMcOfoZ6Uw/viewform?usp=sf_lin).
+Your answers will help our operators submit resource requests to your local cluster of the appropriate size and scale.
 
--   Validate operation with a set of test jobs
+!!! question "Can I change my answers at a later date?"
+    Yes! If you want the OSG to change the size (i.e. CPU, RAM), type (e.g., GPU requests), or number of resource requests,
+    [contact us](mailto:help@opensciencegrid.org) with the FQDN of yoru login host and the details of your changes.
```
Comment (matyasselmeci) [inline docs/compute-element/hosted-ce.md:119]:
```suggestion
    [contact us](mailto:help@opensciencegrid.org) with the FQDN of your login host and the details of your changes.
```

File: docs/compute-element/hosted-ce.md [hunk 2]
```diff
@@ -1,157 +1,185 @@
...
-------------------------------------------------------------------
+![monitoring graphs](img/monitoring_graphs.png)
 
-Many OSG jobs make of use software modules provided by their collaborations or by the OSG User Support team.
-In order to support these jobs without having to install specific software modules on your cluster,
-OSG sites may provide a distributed software repository system called OASIS, built on top of
-[CVMFS](https://cernvm.cern.ch/portal/filesystem).
+To view your contributions, select your site from the `Facility` dropdown of the
+[Payload job summary](https://gracc.opensciencegrid.org/d/000000043/pilot-jobs-summary?orgId=1) dashboard.
+Note that accounting data may take up to 24 hours to display.
```
Comment (matyasselmeci) [inline docs/compute-element/hosted-ce.md:144]: Maybe "up to 24 hours to update"? "24 hours to display" makes it sound like the website is _really slow_.

**Review by matyasselmeci — APPROVED** [review]
Pre-approving; I left two comments but otherwise this is really good!

File: docs/compute-element/hosted-ce.md (hunk 2 above)
Comment (brianhlin) [reply docs/compute-element/hosted-ce.md:145]: Sure, sounds good. Though I think @djw8605 mentioned on software-discuss that usage should show up after 1 hr

Comment (djw8605) [reply docs/compute-element/hosted-ce.md:145]: New registrations take up to 24 hours to show up in GRACC.  Topology is only read every 24 hours by the summarizing process.  But, the usage should show up in ~30 minutes, if the site was already registered 24+ hours ago.

**Comment by brianhlin** [comment]
Got offline reviews from @lmichael107 and @jdost321. Overriding unrelated CI failures and merging.

**Pull request merged**